tests/utils/log/test_log_reader.py

```diff
@@ -199,6 +199,7 @@ def test_read_log_stream_should_support_multiple_chunks(self, mock_read):
         mock_read.side_effect = [first_return, second_return, third_return, fourth_return]
 
         task_log_reader = TaskLogReader()
+        self.ti.state = State.SUCCESS
```

Comment:
Better to use `TaskInstanceState.SUCCESS` here

Reply:
Thanks, applied